lib/assets/javascripts/twine.js.coffee

```diff
@@ -315,7 +315,7 @@ setupAttributeBinding('innerHTML', 'unsafe-html')
 
 preventDefaultForEvent = (event) ->
   (event.type == 'submit' || event.currentTarget.nodeName.toLowerCase() == 'a') &&
-  Twine.getAttribute(event.currentTarget, 'allow-default') != '1'
+  Twine.getAttribute(event.currentTarget, 'allow-default') in ['false', false, 0, undefined, null]
```

Comment:
Yeah I'm wondering if we have any spots where we're doing `allow-default="false"` falsey type things? Isn't this something people only add on after-the-fact to allow the default?